comm.go

```diff
@@ -428,6 +428,12 @@ func (com *Communication) estimateTpb(tpbChan chan<- int) {
 func (com *Communication) Communicate(txCurve map[int32]*Row, miner *Miner, actors []*Actor) {
 	defer com.wg.Done()
 
+	go func() {
```

Comment:
OK, I see that `generate` is blocking, unlike `setgenerate` so actually we could do away with the notifications handlers to wait for the miner. Let me check that.